### DISCUSSION TITLE
Transloco inside custom angular lib (not monorepo environment)

### DISCUSSION THREAD

**Discussion by dionatan-g** [ngneat/transloco#452] [General]
Hello guys!
Im looking for a working example of transloco inside a custom angular lib (not on a monorepo). I would like to handle all the translation inside the lib itself by having a dedicated i18n folder on it, also on some inputs, i would like to let the user pass in translation keys from the "user side" i18n, not the lib.

```
 toolbarActionButtons: ToolbarActionButton[] = [
    {
      label: 'actionButton.label',
      size: ActionButtonSizeEnum.small,
      icon: 'icon-upload-button',
      event: () => console.log('action2'),
    }
  ];
```

This is an example of a input to pass to the lib itself, the label is a key from the user i18n, not the lib, but its the lib that translates it.

**In short:** Let the user pass in keys from the user side i18n, also, let the lib itself have dedicated i18n to internal translation.
**PS:** My lib doesn't have a root module, every component has its own module so I can consume individually.
